packages/order/src/migrations/Migration20240219102530.ts

```diff
@@ -0,0 +1,462 @@
+                  'canceled'
+              )
+          ) NOT NULL DEFAULT 'pending',
+          "internal_note" text NULL,
+          "created_by" text NOT NULL,
+          "requested_by" text NULL,
+          "requested_at" timestamptz NULL,
+          "confirmed_by" text NULL,
+          "confirmed_at" timestamptz NULL,
+          "declined_by" text NULL,
+          "declined_reason" text NULL,
+          "metadata" jsonb NULL,
+          "declined_at" timestamptz NULL,
+          "canceled_by" text NULL,
+          "canceled_at" timestamptz NULL,
+          "created_at" timestamptz NOT NULL DEFAULT now(),
+          "updated_at" timestamptz NOT NULL DEFAULT now(),
+          CONSTRAINT "order_change_pkey" PRIMARY KEY ("id")
+      );
+
+      CREATE INDEX IF NOT EXISTS "IDX_order_change_order_id" ON "order_change" (
+          order_id
+      );
+
+      CREATE INDEX IF NOT EXISTS "IDX_order_change_status" ON "order_change" (status);
+
+      CREATE TABLE IF NOT EXISTS "order_change_action" (
+          "id" TEXT NOT NULL,
+          "order_change_id" TEXT NOT NULL,
+          "reference" TEXT NOT NULL,
+          "reference_id" TEXT NOT NULL,
+          "action" JSONB NOT NULL,
+          "metadata" JSONB NULL,
+          "internal_note" TEXT NULL,
+          "created_at" TIMESTAMPTZ NOT NULL DEFAULT Now(),
+          "updated_at" TIMESTAMPTZ NOT NULL DEFAULT Now(),
+          CONSTRAINT "order_change_action_pkey" PRIMARY KEY ("id")
+      );
+
+      CREATE INDEX IF NOT EXISTS "IDX_order_change_action_order_change_id" ON "order_change_action" (
+          order_change_id
+      );
+
+      CREATE INDEX IF NOT EXISTS "IDX_order_change_action_reference_id" ON "order_change_action" (
+          reference_id
+      );
+
+      CREATE TABLE IF NOT EXISTS "order_detail" (
+          "id" TEXT NOT NULL,
+          "order_id" TEXT NOT NULL,
+          "version" INTEGER NOT NULL,
+          "item_id" TEXT NOT NULL,
+          "quantity" NUMERIC NOT NULL,
+          "raw_quantity" JSONB NOT NULL,
+          "fulfilled_quantity" NUMERIC NOT NULL,
+          "raw_fulfilled_quantity" JSONB NOT NULL,
+          "shipped_quantity" NUMERIC NOT NULL,
+          "raw_shipped_quantity" JSONB NOT NULL,
+          "return_requested_quantity" NUMERIC NOT NULL,
+          "raw_return_requested_quantity" JSONB NOT NULL,
+          "return_received_quantity" NUMERIC NOT NULL,
+          "raw_return_received_quantity" JSONB NOT NULL,
+          "return_dismissed_quantity" NUMERIC NOT NULL,
+          "raw_return_dismissed_quantity" JSONB NOT NULL,
+          "written_off_quantity" NUMERIC NOT NULL,
+          "raw_written_off_quantity" JSONB NOT NULL,
+          "summary" JSONB NOT NULL,
+          "created_at" TIMESTAMPTZ NOT NULL DEFAULT Now(),
+          "updated_at" TIMESTAMPTZ NOT NULL DEFAULT Now(),
+          CONSTRAINT "order_detail_pkey" PRIMARY KEY ("id")
+      );
+
+      CREATE INDEX IF NOT EXISTS "IDX_order_detail_id" ON "order_detail" (id);
```

Comment:
I don't think this is needed, since Postgres will create an index for the primary key.

Reply:
Indeed. Before the PK was a composite key, and the ID a regular index.. I inverted it but didn't remove this index.